lib/pdk/tests/unit.rb

```diff
@@ -6,9 +6,13 @@
 module PDK
   module Test
     class Unit
-      def self.cmd(_tests)
+      def self.cmd(_tests, opts = {})
         # TODO: test selection
-        [File.join(PDK::Util.module_root, 'bin', 'rake'), 'spec']
+        if opts.key?(:parallel)
+          [File.join(PDK::Util.module_root, 'bin', 'rake'), 'parallel']
```

Comment:
This will need to change once puppetlabs_spec_helper is updated. Added the updated `parallel_spec` task to the module's Rakefile as `parallel` to avoid name conflicts.